erpnext/accounts/doctype/payment_request/payment_request.py

```diff
@@ -147,28 +158,41 @@ def validate_subscription_details(self):
 					).format(self.grand_total, amount)
 				)
 
-	def on_submit(self):
-		if self.payment_request_type == "Outward":
-			self.db_set("status", "Initiated")
-			return
-		elif self.payment_request_type == "Inward":
-			self.db_set("status", "Requested")
-
-		send_mail = self.payment_gateway_validation() if self.payment_gateway else None
```

Comment:
This may still be needed. Also check if what you added in test case is required.